src/python/pants/option/global_options.py

```diff
@@ -651,7 +651,7 @@ def register_bootstrap_options(cls, register):
             "--process-execution-cache-namespace",
             advanced=True,
             type=str,
-            default="",
+            default=DEFAULT_EXECUTION_OPTIONS.process_execution_cache_namespace,
```

Comment:
Presumably unrelated?

Reply:
Without this, the readme page for global options causing a 500 error because there's an empty `<code></code>` block. Explained in 9398f64702e6889fb7b2ef513fcbccab26eedb5d.